liquibase-standard/src/main/java/liquibase/util/ObjectMethods.java

```diff
@@ -0,0 +1,49 @@
+package liquibase.util;
+
+import static java.util.Locale.ROOT;
+import static java.util.stream.Collectors.toMap;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BinaryOperator;
+
+final class ObjectMethods {
+
+  private static final BinaryOperator<Method> ignoreMultipleMethodsWithSameName = (first, second) -> first;
+  private final Map<String, Method> writeMethods;
+  private final Map<String, Method> readMethods;
+
+  ObjectMethods(Class klass) {
+    this(klass.getMethods());
+  }
+
+  ObjectMethods(Method[] methods) {
+    readMethods = new HashMap<>(methods.length);
```

Comment:
what does it mean `parametersCount = 0` / `1`?

Reply:
Look at constructor:
```java
    find(methods, 0, "get");
    find(methods, 0, "is");
    find(methods, 1, "set");
```

`get` an `is` methods have 0 parameters (they are called "getters"), 
while `set` methods have 1 parameter (they are called "setters).

Reply:
I know what getters/setters are but I didn't realize it was about the number of arguments each of these methods type could have. I thought it could be another filtering regarding types.

It's clear now!

Thank you!